docs/translations/api-docs/data-grid/data-grid-premium.json

```diff
@@ -942,7 +942,7 @@
       "typeDescriptions": {}
     },
     "throttleRowsMs": {
-      "description": "If positive, the Grid will throttle updates coming from <code>apiRef.current.updateRows</code> and <code>apiRef.current.setRows</code>. It can be useful if you have a high update rate but do not want to do heavy work like filtering / sorting or rendering on each  individual update.",
```

Comment:
We always use `grid` in the other props JSDoc

Reply:
Was going with Data Grid here ever considered? 🤔 
https://www.notion.so/mui-org/Style-conventions-for-MUI-components-and-products-dff7d2d7d1ba4a4abd47f48cf345b800

Reply:
But I do agree `Data Grid` it probably the way to go